server/queue.go

```diff
@@ -170,13 +170,18 @@ func PushLogs(c *gin.Context) {
 	// TODO (bradrydzewski) change this interface to accept an io.Reader
 	// instead of a byte array so that we can buffer the write and so that
 	// we avoid unnecessary copies of the data in memory.
-	logs, err := ioutil.ReadAll(io.LimitReader(c.Request.Body, 5000000)) //5MB
-	defer c.Request.Body.Close()
-	if err != nil {
-		c.Fail(500, err)
-		return
-	}
-	err = store.SetLogs(repo.FullName, bnum, tnum, logs)
+	var maxBuffToRead int64 = 5000000 // 5MB.
```

Comment:
in this case we can use `const` instead of `var`. Note that magic numbers aren't necessarily frowned upon in Go, but I think in this case it is fine